vendor/modules.txt

```diff
@@ -10,6 +10,11 @@ github.com/GehirnInc/crypt
 github.com/GehirnInc/crypt/common
```

Comment:
were there no other additions to the `vendor` directory?

Reply:
Added vendor files in most recent commit